package com.dearbear.exceptionStudy;

public class ExceptionInfo {
	public static void main(String args[]) {
		ExceptionInfo eTest = new ExceptionInfo();
		eTest.cal(-1, -1);
	}
	public void	cal(int x, int y)throws MyException {
		if (x < 0 || y < 0) {
			throw new MyException("不可使用负数");
		}
	}
}

class MyException extends Exception{
	public MyException(String ErrorMessage){
		super(ErrorMessage);
	}
}